### PR TITLE
Milestone 6: Views

### PR DESCRIPTION
**Pull request by skabeo**
In this pull request I;

- Implemented the sneak peak design
- Define the index and show methods in the `user_controller.rb` file to return specific data
- Define the index and show methods in the `post_controller.rb` file to return specific data
- Added styles to each view file
- Added a method in the post method to retrieve all comments from the database
- Setup clickable links to various route in views file

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,3 @@
-<h2>Here is a list of users</h2>
+<% @users.each do |user| %>
+  <%= render 'reuse/user_profile', user: user %>
+<% end %>
```
Comment (abdulhamiid) [inline app/views/users/index.html.erb:3]: Good job using `app/views/users/_user_profile.html.erb`  instead of  `@objects.each`for a better practice 👍

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @skabeo 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Use preprocessed HTML files with embedded Ruby code.
- ✔️ Use layouts and templates for shared content.

## Required Changes ♻️
_Please kindly check the comments under the review._

Kindly fix the broken link ☺

image

image

Let's avoid broken links in our readme for a professional repository


## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (skabeo) [reply app/views/users/index.html.erb:3]: Hello @abdulhamiid 

Is it implying i used the right approach or i should use @objects.each?

**Review by DeliceLydia — APPROVED** [review]
## Hi @skabeo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- addressed changes



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (abdulhamiid) [reply app/views/users/index.html.erb:3]: Hi @skabeo, you have already used a good approach 👍

**Pull request merged**